v0/destinations/yahoo_dsp/transform.js

```diff
@@ -0,0 +1,197 @@
+const sha256 = require("sha256");
+const { BASE_ENDPOINT, ENDPOINTS } = require("./config");
+const {
+  defaultRequestConfig,
+  CustomError,
+  isDefinedAndNotNullAndNotEmpty,
+  defaultPutRequestConfig,
+  removeUndefinedAndNullValues,
+  getSuccessRespEvents,
+  getErrorRespEvents
+} = require("../../util");
+
+const {
+  getAccessToken,
+  populateIncludes,
+  populateExcludes
+} = require("./util");
+
+const populateIdentifiers = (attributeArray, { Config }) => {
+  const seedList = [];
+  const { audienceType } = Config;
+  const { hashRequired } = Config;
+  let listType;
+  if (isDefinedAndNotNullAndNotEmpty(attributeArray)) {
+    // traversing through every element in the add array
+    attributeArray.forEach(element => {
+      const keys = Object.keys(element);
+      keys.forEach(key => {
+        if (key === audienceType) {
+          listType = audienceType;
```

Comment:
What is purpose of listype here after this condition is met?

Reply:
`listType` is not used later?

Reply:
No